staging/src/k8s.io/dynamic-resource-allocation/structured/allocator_test.go

```diff
@@ -3325,6 +3356,40 @@ func TestAllocator(t *testing.T) {
 				deviceAllocationResult(req0, driverA, pool1, device1, false),
 			)},
 		},
+		"device-binding-conditions": {
```

Comment:
We should add tests here to make sure this feature works with the partitionable devices feature. And probably some of the other features like device taints and prioritized list too unless we know they are orthogonal

Reply:
Thank you. Can I resolve this for now?